_posts/2020-02-01-rom-and-dry-showcase-part-1.markdown

```diff
@@ -217,12 +217,12 @@ Then the migration has been successfully applied.
 
 In order to get data into and out of database tables with ROM, we need to create something called a _repository_. A repository is a class that is used to define a clear API between your database and your application.
 
-To create one of these, we'll create a new file inside a new directory structure at `lib/bix/repos/user_repo.rb`:
+To create one of these, we'll create a new file inside a new directory structure at `lib/bix/repos/user.rb`:
```

Comment:
I want to leave this as `user_repo` because later on when we import it with `dry-auto_inject` it'll make it available as a variable called `user_repo`, not `user`.

Reply:
Sure thing!, in the code samples it's mostly `Bix::Repos::User` though, also in part 2 (haven't been up to part 3 ;). Will have to update the code samples to be `Bix::Repos::UserRepo` as well then.